presto-native-execution/presto_cpp/main/connectors/arrow_flight/tests/ArrowFlightConnectorTlsTest.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "arrow/flight/types.h"
+#include "arrow/testing/gtest_util.h"
+#include "folly/init/Init.h"
+#include "glog/logging.h"
+#include "gtest/gtest.h"
+#include "presto_cpp/main/connectors/arrow_flight/ArrowFlightConnector.h"
+#include "presto_cpp/main/connectors/arrow_flight/tests/utils/FlightConnectorTestBase.h"
+#include "presto_cpp/main/connectors/arrow_flight/tests/utils/FlightPlanBuilder.h"
+#include "presto_cpp/main/connectors/arrow_flight/tests/utils/Utils.h"
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/common/config/Config.h"
+#include "velox/exec/tests/utils/AssertQueryBuilder.h"
+#include "velox/exec/tests/utils/OperatorTestBase.h"
+
+namespace facebook::presto::connector::arrow_flight::test {
+
+namespace {
+
+namespace velox = facebook::velox;
```

Comment:
Haven't seen this style of shorthand for namespaces in Prestissimo code. Can these be removed ?